packages/input/src/index.js

```diff
@@ -9,7 +9,7 @@ const StyledContainer = styled.div`
 
 const StyledLabel = styled.label`
     color: #868686;
-    font-family: 'Fira Sans';
+    font-family: -apple-system, BlinkMacSystemFont;
```

Comment:
TBH I'm not sure why I was using a Google Font before. This makes more sense.

Reply:
Do we need something more inclusive and not just Mac focused?

https://github.com/twbs/bootstrap/blob/5512dde203cc3719bac20fb59aa269b99986cfee/scss/_variables.scss#L386

Reply:
Not a bad idea. I'll update!